docs/internals/data-structures.rst

```diff
@@ -88,7 +88,10 @@ Segments
 ~~~~~~~~
 
 Objects referenced by a key are stored inline in files (`segments`) of approx.
-500 MB size in numbered subdirectories of ``repo/data``.
+500 MB size in numbered subdirectories of ``repo/data``. The number of segments
+per directory is controlled by the value of ``segments_per_dir``. If you change
+this value in a non-empty repository, you may also need to relocate the segment
+files manually.
```

Comment:
I did not know about 1.1.x new features until yesterday :-) My Linux distribution (Arch) has only the stable 1.0.10 version. My repositories was created with 10000 s_p_d. When I tried to change s_p_d from 10000 down to 1000, I encountered FileNotFound. That's why I opened this PR in the first place.